src/telegram/commands/answer.rs

```diff
@@ -0,0 +1,124 @@
+use std::hash::{Hash, Hasher};
+use anyhow::{Context};
+use chrono::{DateTime, Utc};
+use rand::SeedableRng;
+use rand::seq::SliceRandom;
+use teloxide::Bot;
+use teloxide::payloads::SendMessageSetters;
+use teloxide::prelude::{Message, Requester};
+use crate::telegram::commands::{CommandResult};
+
+// Cerchiamo di tenere bilanciate le tre colonne, o almeno le prime due.
+// Se avete un'idea ma metterebbe troppe opzioni in un'unica categoria, mettetela sotto commento.
+const ANSWERS: [&str; 60] = [
+	// risposte "sì": 20
+    "🔵 Sì.",
+    "🔵 Decisamente sì!",
+    "🔵 Uhm, secondo me sì.",
+    "🔵 Sì! Sì! SÌ!",
+    "🔵 Yup.",
+    "🔵 Direi proprio di sì.",
+    "🔵 Assolutamente sì.",
+    "🔵 Ma certo!",
+    "🔵 Esatto!",
+    "🔵 Senz'altro!",
+    "🔵 Ovviamente.",
+    "🔵 Questa domanda ha risposta affermativa.",
+    "🔵 Hell yeah.",
+    "🔵 YES! YES! YES!",
+    "🔵 yusssssss",
+    "🔵 Non vedo perchè no",
+    "🔵 Ha senso, ha perfettamente senso, nulla da obiettare, ha senso.",
+    "🔵 Yos!",
+    "🔵 Sì, ma tienilo segreto...",
+    "🔵 [RADIO] Affermativo.",
+
+    // risposte "no": 20
+    "❌ No.",
+    "❌ Decisamente no!",
+    "❌ Uhm, secondo me sì. No, aspetta, ci ho ripensato. È un no.",
+    "❌ No, no, e ancora NO!",
+    "❌ Nope.",
+    "❌ Direi proprio di no.",
+    "❌ Assolutamente no.",
+    "❌ Certo che no!",
+    "❌ Neanche per idea!",
+    "❌ Neanche per sogno!",
+    "❌ Niente affatto!",
+    "❌ Questa domanda ha risposta negativa.",
+    "❌ Hell no.",
+    "❌ NO! NO! NO!",
+    "❌ lolno",
+    "❌ NEIN NEIN NEIN NEIN",
+    "❌ Delet dis",
+    "❌ Nopety nope!",
+    "❌ No, ma tienilo segreto.",
+    "❌ [RADIO] Negativo.",
+
+    // risposte "boh": 20
+    "❔ Boh.",
+    "❔ E io che ne so?!",
+    "❔ Non so proprio rispondere.",
+    "❔ Non lo so...",
+    "❔ Mi avvalgo della facoltà di non rispondere.",
+    "❔ Non parlerò senza il mio avvocato!",
+    "❔ Dunno.",
+    "❔ Perché lo chiedi a me?",
+    "❔ Ah, non lo so io!",
+    r"❔ ¯\_(ツ)_/¯",
```

Comment:
Per rendere più evidente le raw strings mi piace usare la sintassi con i cancelletti:

```suggestion
    r#"❔ ¯\_(ツ)_/¯"#,
```

Così è più facile vedela a prima vista, e in più si possono mettere anche `"` dentro la stringa stessa!